### PR TITLE
Unit tests for ActivityCollector configuration validators

### PR DESCRIPTION
**Pull request by liljenback**
## Description

Testing the configuration validators introduced in ActivityCollector createConfigValidators

## Related Issue

CORE-36750

## Motivation and Context

Unit test all the files!

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html) or I'm an Adobe employee.
- [x] I have submitted a [documentation](https://github.com/AdobeDocs/alloy-docs) pull request or no changes are needed.
- [x] I have made any necessary test changes and all tests pass.
- [x] I have run the Sandbox successfully.

### CODE REVIEW

File: test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js [hunk 1]
```diff
@@ -12,3 +12,53 @@ governing permissions and limitations under the License.
 
 // eslint-disable-next-line no-unused-vars
 import createConfigValidators from "../../../../../src/components/ActivityCollector/createConfigValidators";
+import createConfig from "../../../../../src/core/createConfig";
+import { config } from "rxjs";
```
Comment (jfkhoury) [inline test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js]: What is this little guy doing here? :)

File: test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js [hunk 2]
```diff
@@ -12,3 +12,53 @@ governing permissions and limitations under the License.
...
+    },
+    {
+      clickCollectionEnabled: false,
+      downloadLinkQualifier: ""
+    }
+  ].forEach((cfg, i) => {
+    it(`validates configuration (${i})`, () => {
+      const configObj = createConfig(cfg);
+      configObj.addValidators(createConfigValidators());
+      configObj.validate();
```
Comment (jfkhoury) [inline test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js]: Do we need an `expect` here?

Comment (liljenback) [reply test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js]: It throws which cause the test to fail.

File: test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js (hunk 1 above)
Comment (liljenback) [reply test/unit/specs/components/ActivityCollector/createConfigValidators.spec.js]: Hmm, good question

**Pull request merged**